src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.15.0-Nightly20160212
+set autoRestVersion=0.16.0-Nightly20160310
```

Comment:
@amarzavery 
I think the relative path shall be abstracted and handled by the program, not hard coded in the spec file.